kedro-datasets/kedro_datasets/matplotlib/matplotlib_writer.py

```diff
@@ -44,8 +44,9 @@ class MatplotlibWriter(
         >>> from kedro_datasets.matplotlib import MatplotlibWriter
         >>>
         >>> fig = plt.figure()
-        >>> plt.plot([1, 2, 3])
-        >>> plot_writer = MatplotlibWriter(filepath="data/08_reporting/output_plot.png")
+        >>> plt.plot([1, 2, 3])  # doctest: +ELLIPSIS
+        [<matplotlib.lines.Line2D object at 0x...>]
```

Comment:
It's possible to avoid the ellipses by sticking a semicolon on the end of `plt.plot()` invocations; however, I'd rather not do something like that just to hide the output from the user (think it's more likely to be confusing than this, if the user were to copy an example).